src/mas/devops/tekton.py

```diff
@@ -402,6 +402,18 @@ def launchInstallPipeline(dynClient: DynamicClient, params: dict) -> str:
     return pipelineURL
 
 
+def launchInstallPipelineForAiservice(dynClient: DynamicClient, params: dict) -> str:
+    """
+    Create a PipelineRun to install the chosen MAS instance (and selected dependencies)
+    """
+    instanceId = params["mas_instance_id"]
+    namespace = f"mas-{instanceId}-pipelines"
+    timestamp = launchPipelineRun(dynClient, namespace, "pipelinerun-aiservice-install", params)
+
+    pipelineURL = f"{getConsoleURL(dynClient)}/k8s/ns/mas-{instanceId}-pipelines/tekton.dev~v1beta1~PipelineRun/{instanceId}-install-{timestamp}"
```

Comment:
Not sure why the mas_instance_id is related to this.

Reply:
@Bhautik-Vala the issue is that the variable name is wrong. You would be using the value of this to be the aiservice instance id not the mas instanceid.